exampleCourse/questions/demo/annotated/MarkovChainGroupActivity/MarkovChains-Gambler/serverFilesQuestion/graph.py

```diff
@@ -24,7 +24,7 @@ def get_bytes(self):
 
 
 def _draw_adj_matrix(
-    mat, mat_label, show_weights, round_digits, directed, layout="dot"
+    mat, mat_label, *, show_weights: bool, round_digits, directed, layout="dot"
```

Comment:
I don't think this code is even typechecked?

Reply:
It is not -- I added the `: bool` annotation to trigger ruff linting the boolean trap rule on this.

Reply:
I think this is a little unnecessary for the example course, since we don't include type annotations in these examples.